src/app/api/redirect/route.ts

```diff
@@ -0,0 +1,51 @@
+import { NextRequest, NextResponse } from "next/server";
```

Comment:
This API was added for redirect purpose so it will be kept or removed based on how BE transak integration works.